source/modules/validators/validate.py

```diff
@@ -242,3 +246,14 @@ def verify_task_permissions(self, selected_items: dict, user_groups: list, confi
                                 return True
         self.logger.debug("Task forbidden")
         return False
+
+    @staticmethod
+    def expand_option_name(option: str):
```

Comment:
idk... it creates more names

Reply:
Could be confusing for developer/tester but it improves user interface.